docs/content/contrib-docs.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Contributing Docs
+kind: contrib
+weight: 2
+---
+
+Contributing to our docs is one of the best ways to get started contributing to the OPA project. The OPA docs are one of the first places most people go for help, so any changes are impactful for the community. Thanks in advance for contributing to the docs.
+
+## Docs Layout
+
+* [Core Docs](https://www.openpolicyagent.org/docs/latest/) and reference materials for OPA and Rego
+* [Kubernetes](../kubernetes-introduction/) Admission Control Docs
+* [Envoy](../envoy-introduction/) docs for the OPA-Envoy Plugin
+* Other Use Cases like [Docker](../docker-authorization/), [Kafka](../kafka-authorization/), and [Terraform](../terraform/)
+* [Operations](../configuration/), configuration, and deployment docs for OPA   
+* [Management APIs](../management-introduction/) docs to control OPA at scale
+* [Contributing](../contributing/) to the OPA project docs
+* Miscellaneous docs like [WebAssembly](../wasm/), [OAuth2](../oauth-oidc/), and [IDE Support](../editor-and-ide-support/)
+* [Support](https://www.openpolicyagent.org/support) for Enterprise and Commercial deployments
+
+Sub-project docs
+* [Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/docs/) docs for Kubernetes Admission Control
+* [Conftest](https://www.conftest.dev/) docs for validating your structured configuration data, like YAML and HCL files 
```

Comment:
This section is going to quickly get out of date as we add new sections, refactor existing sections, etc. I think we could drop this without losing much.

Reply:
I could see it being useful to link off to the other subprojects docs here.